### ISSUE TITLE
Division by 0

### ISSUE THREAD

**Issue by EmmaRenauld** [scilus/scilpy#884]
In the pytest of scil_dti_metrics:

RuntimeWarning: divide by zero encountered in log
    D[..., 6] = -np.log(b0)

This originates from dipy/reconst/dti.py:2120, which is in method lower_triangular, which is called through our own line 215:     tenfit = tenmodel.fit(data, mask)

**Comment by AntoineTheb** [comment]
Fun fact: debugging warnings can be made easier by making pytest crash on warnings https://stackoverflow.com/a/59383021
